From 866fc90b224723260bf78136ea19924a45044c64 Mon Sep 17 00:00:00 2001
From: Hongmei Gou <h-gou@ti.com>
Date: Fri, 19 Jun 2015 15:32:01 -0400
Subject: [PATCH] UAPI: Increasing the max FRAME number to 128.

This is required because on TI SoC, the IVA_HD can consume as many as
32 buffers for some H.264 streams. And each buffer can contain plane/field
data as Y-top, Y-bottom, UV-top and UV-bottom. Each field will be treated
as a separate FRAME.

This patch comes from glsdk kernel tree:

commit 275dfdd03867f8b402942304008fc7bc4ca35933
Author: Somnath Mukherjee <somnath@ti.com>
Date:   Mon Mar 24 21:04:56 2014 +0530

Change-Id: Ibc004ad6f5c7c92a13cd060a7242a8edaa6b55e3
Signed-off-by: Somnath Mukherjee <somnath@ti.com>
Signed-off-by: Nikhil Devshatwar <nikhil.nd@ti.com>

Signed-off-by: Hongmei Gou <h-gou@ti.com>
---
 include/uapi/linux/videodev2.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/videodev2.h b/include/uapi/linux/videodev2.h
index d26bcff..2769c62 100644
--- a/include/uapi/linux/videodev2.h
+++ b/include/uapi/linux/videodev2.h
@@ -69,7 +69,7 @@
  * Common stuff for both V4L1 and V4L2
  * Moved from videodev.h
  */
-#define VIDEO_MAX_FRAME               32
+#define VIDEO_MAX_FRAME              128
 #define VIDEO_MAX_PLANES               8
 
 /*
-- 
1.7.9.5

